src/ProjectOrigin.Electricity.Example/Helper.cs

```diff
@@ -0,0 +1,212 @@
+using System.Security.Cryptography;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using ProjectOrigin.Common.V1;
+using ProjectOrigin.Electricity.V1;
+using ProjectOrigin.HierarchicalDeterministicKeys.Interfaces;
+using ProjectOrigin.PedersenCommitment;
+using SimpleBase;
+
+namespace ProjectOrigin.Electricity.Example;
+
+public class Helper
+{
+    static DateTimeOffset Start = new DateTimeOffset(2023, 1, 1, 12, 0, 0, 0, TimeSpan.Zero);
+    private string _area;
+
+    public Helper(string area)
+    {
+        _area = area;
+    }
+
+    public Electricity.V1.ConsumptionIssuedEvent CreateConsumptionIssuedEvent(FederatedStreamId certId, SecretCommitmentInfo commitmentInfo, IPublicKey ownerKey)
+    {
+        var @event = new Electricity.V1.ConsumptionIssuedEvent
+        {
+            CertificateId = certId,
+            Period = new Electricity.V1.DateInterval
+            {
+                Start = Timestamp.FromDateTimeOffset(Start),
+                End = Timestamp.FromDateTimeOffset(Start.AddHours(1))
+            },
+            GridArea = _area,
+            GsrnHash = ByteString.Empty,
+            QuantityCommitment = CommitmentToProto(certId, commitmentInfo),
+            OwnerPublicKey = new Electricity.V1.PublicKey
+            {
+                Content = ByteString.CopyFrom(ownerKey.Export())
+            }
+        };
+
+        return @event;
+    }
+
+    public Electricity.V1.ProductionIssuedEvent CreateProductionIssuedEvent(FederatedStreamId certId, SecretCommitmentInfo commitmentInfo, IPublicKey ownerKey)
+    {
+        var @event = new Electricity.V1.ProductionIssuedEvent
+        {
+            CertificateId = certId,
+            Period = new Electricity.V1.DateInterval
+            {
+                Start = Timestamp.FromDateTimeOffset(Start),
+                End = Timestamp.FromDateTimeOffset(Start.AddHours(1))
+            },
+            GridArea = _area,
+            TechCode = "T010101",
+            FuelCode = "F010101",
+            GsrnHash = ByteString.Empty,
+            QuantityCommitment = CommitmentToProto(certId, commitmentInfo),
+            OwnerPublicKey = new Electricity.V1.PublicKey
+            {
+                Content = ByteString.CopyFrom(ownerKey.Export())
+            }
+        };
+
+        return @event;
+    }
+
+    private static V1.Commitment CommitmentToProto(FederatedStreamId certId, SecretCommitmentInfo commitmentInfo)
```

Comment:
This could be moved to the bottom of the class as the public methods below also use `CommitmentToProto`